mmdet/core/bbox/iou_calculators/iou2d_calculator.py

```diff
@@ -189,7 +198,7 @@ def bbox_overlaps(bboxes1, bboxes2, mode='iou', is_aligned=False, eps=1e-6):
         >>> assert tuple(bbox_overlaps(empty, empty).shape) == (0, 0)
     """
 
-    assert mode in ['iou', 'iof', 'giou'], f'Unsupported mode {mode}'
+    assert mode in ['iou', 'iof', 'giou', 'diou'], f'Unsupported mode {mode}'
```

Comment:
Where did you use diou in ddod?

Reply:
We just used it in our experiment of ddod, not the code now.

Reply:
I think it is better not to add  diou for now if it is not used here.

Reply:
Ok, I will delete the diou code in here.

Reply:
And unit test for diou

Reply:
I have modified it in new push.